packages/design-system/src/components/Banner/Banner.md

```diff
@@ -0,0 +1,167 @@
+### Banner
+
+## Usage
```

Comment:
It would be cool to add some general info regarding component's usage, like @kamilmateusiak [suggested you](https://github.com/livechat/design-system/pull/190#discussion_r435982445) for `Promo`.

Reply:
Waiting for description from bartek :)